administrator/components/com_tags/models/tag.php

```diff
@@ -133,20 +133,9 @@ public function getItem($pk = null)
 			$registry = new Registry($result->urls);
 			$result->urls = $registry->toArray();
 
-			// Convert the created and modified dates to local user time for display in the form.
+			// Convert the modified dates to local user time for display in the form.
```

Comment:
Singular not plural.

```suggestion
			// Convert the modified date to local user time for display in the form.
```